test/fixtures/component-playground/proxies.js

```diff
@@ -29,6 +29,12 @@ const MarkupProxy = React.createClass({
 });
 
 module.exports = _.merge({}, selectedFixture, {
-  proxies: [PropMutatorProxy, MarkupProxy],
+  proxies: [{
+    component: PropMutatorProxy,
+    props: {},
```

Comment:
> What happens if another proxy needs a key named 'store' as well, for completely different reasons? It will use the same value.

To avoid this problem completely, we can make proxies be functions that return components, instead of components. E.g.

```js
proxies: [
  // Default 
  reactCosmosReduxProxy()
  // Custom key
  reactCosmosReduxProxy({ storeKey: '__reduxStore' }),
]
```

And inside the proxy:

```js
const reactCosmosReduxProxy = ({ storeKey }) =>
  class ReduxProxy extends React.Component {
    constructor(fixture) {
      super();
        this.store = createStore((state) => state, fixture[storeKey || 'reduxStore']);
    }
```

Notes:
- We could default to "reduxStore" instead of "store", to minimize such conflicts from the onset. 
- I suggest using `fixture` instead of `props` inside proxies. It sounds unorthodox, but I think is easier to understand. After all, the props of the preview component just happen to be all the fixture keys minutes the *magical* ones (like "state", and now "store", or "reduxStore").
  - Moreover, I think this proxy should omit the redux store key from the fixture when passing it down to children, just like ComponentTree does with "state" key. Children should receive just props (remaining fixture keys after magical ones).